CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/TableViewPane.java

```diff
@@ -827,17 +912,17 @@ private void loadPreferences() {
                 // Populate orderedColumns with full column ThreeTuples corresponding to entires in newVolumnOrder and call updateVisibleColumns
                 // to update table.
                 final List<ThreeTuple<String, Attribute, TableColumn<ObservableList<String>, String>>> orderedColumns
-                    = newColumnOrder.stream()
-                        .map(c ->
-                            {
-                                for (ThreeTuple<String, Attribute, TableColumn<ObservableList<String>, String>> col : columnIndex) {
-                                    if (c.getText().equals(col.getThird().getText())) {
-                                        return col;
+                        = newColumnOrder.stream()
+                                .map(c
+                                        -> {
```

Comment:
I think this was done in the other PR but for readability combine these 3 lines into 1:
`...stream().map(c -> {`

Reply:
Yeah, caused by Alt+Shift+F
Fixed now.